superdesk/activity.py

```diff
@@ -229,7 +229,7 @@ def add_activity(
 
 
 def notify_and_add_activity(
-    activity_name, msg, resource=None, item=None, user_list=None, users_with_enabled_email_notifications=None, **data
+    activity_name, msg, resource=None, item=None, user_list=None, preference_notification_name=None, **data
```

Comment:
why not use the activity_name for the notification settings?

Reply:
I thought the `activity_name`(`item:marked`, `item:unmarked`) is used in emails

Reply:
I see, makes sense then